postgres/datadog_checks/postgres/statements.py

```diff
@@ -163,3 +196,145 @@ def _collect_statement_metrics(self, instance_tags):
                 fn(self, name, val, tags=tags)
 
         self.statement_cache = new_cache
+
+    def _get_new_pg_stat_activity(self, instance_tags=None):
+        start_time = time.time()
+        query = """
+        SELECT * FROM pg_stat_activity
+        WHERE datname IS NOT NULL
```

Comment:
The check is per-db. This should not pull all activity from all dbs. If a customer has a list of all several postgres dbs in their config, this will duplicate samples/exec plans.